src/plugins/postal-savings-rs/__tests__/converters/transactions/cardTransactions.test.ts

```diff
@@ -0,0 +1,131 @@
+import { convertCardTransactions } from '../../../converters'
+
+const input= `
+ <tr>
+    <td align="left">05.04.2024</td>
+    <td align="right">-3.282,63</td>
+    <td align="left">RSD</td>
+    <td align="left">09.04.2024</td>
+    <td align="left">Wolt doo                      </td>
+    <td align="right">4870........1234</td>
+</tr>
+<tr>
+    <td align="left">06.04.2024</td>
+    <td align="right">-370,00</td>
+    <td align="left">RSD</td>
+    <td align="left">09.04.2024</td>
+    <td align="left">LP NOVI CAJ VLADIMIR     NOVI </td>
+    <td align="right">4870........1234</td>
+</tr>
+<tr>
+    <td align="left">07.04.2024</td>
+    <td align="right">500,00</td>
+    <td align="left">EUR</td>
+    <td align="left">10.04.2024</td>
+    <td align="left">REFUND U INOSTRANSTVU         </td>
+    <td align="right">4870........1234</td>
+</tr>
+<tr>
+    <td align="left">08.04.2024</td>
+    <td align="right">-210,00</td>
+    <td align="left">RSD</td>
+    <td align="left">10.04.2024</td>
+    <td align="left">TISHLER TOBACCO AND COFFENOVI </td>
+    <td align="right">4870........1234</td>
+</tr>
+<tr>
+    <td align="left">12.04.2024</td>
+    <td align="right">20,00</td>
+    <td align="left">USD</td>
+    <td align="left">          </td>
+    <td align="left">CHATGPT SUBSCRIPTION     +1415</td>
+    <td align="right">4870........1234</td>
+</tr>
+<tr>
+    <td align="left">13.04.2024</td>
+    <td align="right">10.000,00</td>
+    <td align="left">RSD</td>
+    <td align="left">          </td>
+    <td align="left">ATMBPS KAFE RANDEVU      NOVI </td>
+    <td align="right">4870........1234</td>
+</tr>
+<tr>
+    <td align="left">13.04.2024</td>
+    <td align="right">250,00</td>
+    <td align="left">EUR</td>
+    <td align="left">          </td>
+    <td align="left">REFUND U INOSTRANSTVU         </td>
+    <td align="right">4870........1234</td>
+</tr>
+`
+
+describe('convertCardTransactions', () => {
+    
+    it('converts card transactions', () => {
+        expect(convertCardTransactions(input)).toEqual([
+            {
+                date: new Date('2024-04-13T00:00:00.000'),
+                authorizationDate: null,
+                amount: {
+                    sum: 250.00,
+                    instrument: 'EUR',
+                },
+                merchant: 'REFUND U INOSTRANSTVU',
+            },
+            {
+                date: new Date('2024-04-13T00:00:00.000'),
+                authorizationDate: null,
+                amount: {
+                    sum: -10_000.00,
+                    instrument: 'RSD',
+                },
+                merchant: 'ATMBPS KAFE RANDEVU',
```

Comment:
It looks like all transactions starting with `ATMBPS` or `ATM BPS` can be safely assumed as cash withdrawal.